crates/config/src/config.rs

```diff
@@ -560,10 +560,13 @@ pub struct SyscallCyclesConfig {
     pub sys_get_block_hash_cycles: u64,
     pub sys_recover_account_cycles: u64,
     pub sys_log_cycles: u64,
+    pub sys_bn_add_cycles: u64,
+    pub sys_bn_mul_cycles: u64,
+    pub sys_bn_pairing_cycles: u64,
 }
 
 impl SyscallCyclesConfig {
-    pub fn all_zero() -> Self {
+    pub fn default() -> Self {
```

Comment:
Please modify the default config according to mainnet settings.

Reply:
https://github.com/godwokenrises/godwoken/pull/808/commits/ead4e4faf3b36f0f5e08df9fa51600f774350e4b